### PR TITLE
Release version 0.19

### LINKED ISSUE #322: Release 0.19
All tasks in the milestone are completed, meaning we can now release `v0.19`.

This will unblock https://github.com/libp2p/rust-libp2p/pull/3656 (already updated to latest HEAD) which will unblock our `rust-libp2p` `v0.52` release.

### PR DESCRIPTION
**Pull request by thomaseizinger**
Not sure how the actual release works but this should be all that is necessary I think.

Resolves #322.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,32 @@
+## [](https://github.com/multiformats/rust-multihash/compare/v0.18.0...v) (2023-06-06)
```
Comment (thomaseizinger) [inline CHANGELOG.md:1]:
```suggestion
## [](https://github.com/multiformats/rust-multihash/compare/v0.18.0...v0.19.0) (2023-06-06)
```

**Comment by vmx** [comment]
@thomaseizinger I've expanded the migration section and updated the release flow information.

**Review by vmx — APPROVED** [review]
I've force-pushed the squashed commit with the commit message I would've used when I would've merged it.

@thomaseizinger feel free to merge if if you agree with my changes (or leave a comment) and I'll merge and do a release afterwards.

**Review by thomaseizinger — COMMENTED** [review]
LGTM!

**Comment by vmx** [comment]
Released! Thanks again @thomaseizinger, you did all the hard work of this gigantic release!

**Comment by mxinden** [comment]
Thank you @vmx and @thomaseizinger!

**Pull request merged**